src/libraries/Common/src/Internal/Cryptography/AsymmetricAlgorithmHelpers.Ansi.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static partial class AsymmetricAlgorithmHelpers
+    {
+        // Encodes a EC key as an uncompressed set of concatenated scalars,
+        // optionally including the private key. To omit the private parameter,
+        // "d" must have a length of zero.
+        public static void EncodeToUncompressedAnsiX963Key(
```

Comment:
This code looks familiar :-) https://github.com/dotnet/runtime/pull/38101/files#diff-e2720fa1848661f3d551c342e97a46b9889d7eba67135ab3eae2882cb22990e6R16

Would appreciate a `Co-authored-by` if it isn't too much trouble and it ends up getting used.